src/rpc/rpc-handlers.ts

```diff
@@ -298,6 +275,9 @@ const handleRPC: HandlerFunc = async data => {
             case 'solana':
                 result = await solana.send({method, params, id, network});
                 break;
+            case 'cosmos':
+                result = await keplrSend({method, params, id, network});
+                break;
```

Comment:
Following the comment above, the idea is that, if we end up here, the method is not _special_, meaning it does not require user confirmation. So, it can be sent to the blockchain, in this case, using a keplr client to do so.